src/layouts/DefaultAppBar.js

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Stack from "@mui/material/Stack";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import { ThemeProvider, createTheme } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+
+function DefaultAppBar(label) {
+  return (
+    <Toolbar>
+      <IconButton edge="start" color="inherit" aria-label="menu" sx={{ mr: 2 }}>
+        <MenuIcon />
+      </IconButton>
+      <Typography variant="h6" noWrap component="div" sx={{ flexGrow: 1 }}>
+        {label}
+      </Typography>
+      <Box>
+        <Box sx={{ display: "flex", gap: 2 }}>
+          <Button sx={{ color: "black", backgroundColor: "#bdc3c7" }}>
+            EN
```

Comment:
While we only have 2 languages, I think we could use a for loop here for future extensible.

Reply:
done